libs/partners/google-vertexai/langchain_google_vertexai/chat_models.py

```diff
@@ -141,8 +143,23 @@ def _convert_to_prompt(part: Union[str, Dict]) -> Part:
 
     vertex_messages = []
     for i, message in enumerate(history):
-        if i == 0 and isinstance(message, SystemMessage):
-            raise ValueError("SystemMessages are not yet supported!")
+        if (
+            i == 0
+            and isinstance(message, SystemMessage)
+            and not convert_system_message_to_human
+        ):
+            raise ValueError(
+                """SystemMessages are not yet supported!
+                
+To automatically convert the leading SystemMessage to a HumanMessage,
+set  `convert_system_message_to_human` to True. Example:
+
+llm = ChatVertexAI(model_name="gemini-pro", convert_system_message_to_human=True)
+"""
+            )
+        elif i == 0 and isinstance(message, SystemMessage):
+            raw_system_message = message
```

Comment:
I think we might need a `raw_system_message = None` above

Reply:
(ignore if pulling out of the loop)

Reply:
Added `raw_system_message = None`